tests/base/src/test/java/org/wildfly/security/auth/server/RegexNameRewriterTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2022 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.security.auth.server;
+
+import org.junit.Test;
+import org.wildfly.security.auth.permission.LoginPermission;
+import org.wildfly.security.auth.realm.FileSystemSecurityRealm;
+import org.wildfly.security.auth.util.RegexNameRewriter;
+
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class RegexNameRewriterTest {
```

Comment:
@lvydra Thank you for the PR! 

This specific issue is to add tests for https://github.com/wildfly-security/wildfly-elytron/blob/1.x/auth/util/src/main/java/org/wildfly/security/auth/util/RegexNameValidatingRewriter.java specifically. This commit adds tests for RegexNameRewriter which can be part of issue [ELY-2446](https://issues.redhat.com/browse/ELY-2446). 

You can add new commit with issue ELY-2446 and add new tests for RegexNameRewriter under it. But for ELY-2439 we are looking to add tests for RegexName*Validating*Rewriter.

Reply:
@Skyllarr Thank you, I misread the description.